danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/GitKtx.kt

```diff
@@ -0,0 +1,59 @@
+package systems.danger.kotlin
+
+// extensions over [Git] object
+
+/**
+ * Changed lines in this MR
```

Comment:
Nitpick: I think PR is the more used term.

Reply:
Yea you're right, I even use it in the wrapper class c22ea384100a297780be7b0499ba6687a407e0b6